sql/src/main/scala/org/datasyslab/geosparksql/utils/DataFrameFactory.scala

```diff
@@ -0,0 +1,27 @@
+package org.datasyslab.geosparksql.utils
+import com.vividsolutions.jts.geom.Geometry
+
+import org.apache.hadoop.fs.{FileSystem, Path}
+import org.datasyslab.geospark.formatMapper.shapefileParser.ShapefileReader
+import org.datasyslab.geospark.formatMapper.shapefileParser.parseUtils.dbf.DbfParseUtil
+import org.datasyslab.geospark.spatialRDD.SpatialRDD
+import scala.collection.JavaConversions._
+
```

Comment:
Can you please do the following change 
(1) Create a function called readFieldName()in ShapefileReader.java (a small function similar [readBoundBox](https://github.com/DataSystemsLab/GeoSpark/blob/master/core/src/main/java/org/datasyslab/geospark/formatMapper/shapefileParser/ShapefileReader.java#L126)). This function returns a list/array of strings which are the field name.
(2) Call ShapefileReader.readFieldName in geometryDfFromShapeFile instead of having the following
```
+    val inputPath = new Path(path)
+    val fileSys = FileSystem.get(spark.sparkContext.hadoopConfiguration)
+    val dbfs = fileSys.listStatus(inputPath).filter(i => i.getPath.getName.toLowerCase.contains("dbf"))
+    if(dbfs.length>0) {
+      val inputStream = fileSys.open(dbfs(0).getPath)
+      val dbfParser = new DbfParseUtil
+      dbfParser.parseFileHead(inputStream)
+      val x = dbfParser.getFieldDescriptors
+      val op = "rddshape" +: x.map(x => x.getFieldName).toSeq
```
(3) Add the necessary function call of DataFrameFactory in constructorTestScala it("Read shapefile to DataFrame"). This will run unit test on DataFrameFactory function.